pkg/appmgmt/general/general.go

```diff
@@ -133,22 +133,16 @@ func (os *Manager) getAppMetadata(pod *v1.Pod) (interfaces.ApplicationMetadata,
 			zap.Error(err))
 	}
 	ownerReferences := getOwnerReferences(pod)
-
-	placeholderTimeout, err := utils.GetPlaceholderTimeoutParam(pod)
-	if err != nil {
-		log.Logger().Debug("unable to get placeholder timeout for pod.",
-			zap.String("namespace", pod.Namespace),
-			zap.String("name", pod.Name),
-			zap.Error(err))
-	}
+	schedulingPolicyParams := utils.GetSchedulingPolicyParam(pod)
 	return interfaces.ApplicationMetadata{
 		ApplicationID:           appID,
 		QueueName:               utils.GetQueueNameFromPod(pod),
 		User:                    user,
 		Tags:                    tags,
 		TaskGroups:              taskGroups,
-		PlaceholderTimeoutInSec: placeholderTimeout,
+		PlaceholderTimeoutInSec: schedulingPolicyParams.GetPlaceholderTimeout(),
```

Comment:
instead of having separate parameters `PlaceholderTimeoutInSec` and `SchedulingStyle`, let's use `SchedulingPolicyParameters` field in the `ApplicationMetadata`, this will be more flexible for future changes.